bundles/org.openhab.binding.electroluxappliance/src/main/java/org/openhab/binding/electroluxappliance/internal/api/ElectroluxGroupAPI.java

```diff
@@ -102,7 +122,7 @@ public boolean refresh(Map<String, ApplianceDTO> electroluxApplianceThings, bool
                             ApplianceStateDTO applianceState = gson.fromJson(jsonApplianceState,
                                     AirPurifierStateDTO.class);
                             if (applianceState != null) {
-                                dto.setApplianceState(applianceState);
+                                dto.setApplianceState(applianceState, retrievalTs);
```

Comment:
Why is only AIR_PURIFIER using the new timestamp method?

Reply:
Good question, I must have missed a commit on one of my pushes, I just updated it again so all three types now have the poll time on the PR request.